examples_pytorch/example4.py

```diff
@@ -2,110 +2,126 @@
 Example 4. Finding camera parameters.
```

Comment:
All the examples for torch except this works. This one does not converge. Can you take a look into it, please?

Reply:
Fixed. Can you run it again, please?